### ISSUE TITLE
my loss is really weird...

### ISSUE THREAD

**Issue by mqjinwon** [STVIR/pysot#265]
my loss didn't decrease and loss value is huge. what did I wrong?

I made YOLOv3 backbone and put pre-trained weight file... so you can ignore "BACKBONE" parts

my config file is below

additionally,  after passing backbone network, zf : torch.Size([24, 128, 16, 16]) and xf : torch.Size([24, 128, 32, 32])

```
META_ARC: "siamrpn_r50_l234_dwxcorr_lt"

BACKBONE:
    TYPE: "resnet50"
    KWARGS:
        used_layers: [2, 3, 4]

ADJUST:
    ADJUST: false

RPN:
    TYPE: 'UPChannelRPN'
    KWARGS:
        anchor_num: 5
        feature_in: 128

MASK:
    MASK: False

ANCHOR:
    STRIDE: 8
    RATIOS: [0.33, 0.5, 1, 2, 3]
    SCALES: [8]
    ANCHOR_NUM: 5

TRACK:
    TYPE: 'SiamRPNTracker'
    PENALTY_K: 0.01
    WINDOW_INFLUENCE: 0.40
    LR: 0.38
    EXEMPLAR_SIZE: 127
    INSTANCE_SIZE: 255
    BASE_SIZE: 8
    CONTEXT_AMOUNT: 0.5

TRAIN:
    EPOCH: 20
    START_EPOCH: 0
    BATCH_SIZE: 24
    BASE_LR: 0.005
    CLS_WEIGHT: 1.0
    LOC_WEIGHT: 1.2
    RESUME: ''

    LR:
        TYPE: 'log'
        KWARGS:
            start_lr: 0.005
            end_lr: 0.0005
    LR_WARMUP:
        TYPE: 'step'
        EPOCH: 5
        KWARGS:
            start_lr: 0.001
            end_lr: 0.005
            step: 1
 META_ARC: "siamrpn_r50_l234_dwxcorr_lt"

BACKBONE:
    TYPE: "resnet50"
    KWARGS:
        used_layers: [2, 3, 4]

ADJUST:
    ADJUST: false

RPN:
    TYPE: 'UPChannelRPN'
    KWARGS:
        anchor_num: 5
        feature_in: 128

MASK:
    MASK: False

ANCHOR:
    STRIDE: 8
    RATIOS: [0.33, 0.5, 1, 2, 3]
    SCALES: [8]
    ANCHOR_NUM: 5

TRACK:
    TYPE: 'SiamRPNTracker'
    PENALTY_K: 0.01
    WINDOW_INFLUENCE: 0.40
    LR: 0.38
    EXEMPLAR_SIZE: 127
    INSTANCE_SIZE: 255
    BASE_SIZE: 8
    CONTEXT_AMOUNT: 0.5

TRAIN:
    EPOCH: 20
    START_EPOCH: 0
    BATCH_SIZE: 24
    BASE_LR: 0.005
    CLS_WEIGHT: 1.0
    LOC_WEIGHT: 1.2
    RESUME: ''

    LR:
        TYPE: 'log'
        KWARGS:
            start_lr: 0.005
            end_lr: 0.0005
    LR_WARMUP:
        TYPE: 'step'
        EPOCH: 5
        KWARGS:
            start_lr: 0.001
            end_lr: 0.005
            step: 1
```

![image](https://user-images.githubusercontent.com/42585679/71070270-c53f0a80-21bd-11ea-8062-67bb22a3fa14.png)

**Comment by yc-zc** [comment]
So... what is your backbone? resnet50 or yolov3?
It sounds pretty weird to use a yolov3 to extract feature, have you succeeded in anywhere else?

**Comment by mqjinwon** [comment]
> So... what is your backbone? resnet50 or yolov3?
> It sounds pretty weird to use a yolov3 to extract feature, have you succeeded in anywhere else?

after getting config.yaml file, I changed the backbone. below is my code.
![image](https://user-images.githubusercontent.com/42585679/71137092-4e038800-224a-11ea-833e-e9124b9df980.png)

actually I found problem, "UPChannelRPN" is something wrong. so I changed it to 'DepthwiseRPN'.
now training is working.
![image](https://user-images.githubusercontent.com/42585679/71137277-e437ae00-224a-11ea-91ad-cc8428094e2e.png)

**Comment by yc-zc** [comment]
> > So... what is your backbone? resnet50 or yolov3?
> > It sounds pretty weird to use a yolov3 to extract feature, have you succeeded in anywhere else?
> 
> after getting config.yaml file, I changed the backbone. below is my code.
> ![image](https://user-images.githubusercontent.com/42585679/71137092-4e038800-224a-11ea-833e-e9124b9df980.png)
> 
> actually I found problem, "UPChannelRPN" is something wrong. so I changed it to 'DepthwiseRPN'.
> now training is working.
> ![image](https://user-images.githubusercontent.com/42585679/71137277-e437ae00-224a-11ea-91ad-cc8428094e2e.png)

Glad to hear you have resolved your issues. 
I'm closing it for now.
